### DISCUSSION TITLE
Is there anyway to limit the resource that one node can use?

### DISCUSSION THREAD

**Discussion by PengfeiM** [coreemu/core#559] [Ideas]
I wonder if there is anyway to limit the resource that one node can use because I think it is appropriate to have a limited capability. 
Furthermore, I suggest to add this function in the configuration of the node so that we can set the nodes' capability separately.
For example, set a limitation of CPUs, RAM and storage that one node can use or the bandwidth upper limit of ethernet and WLAN.

**Comment by bharnden** [comment]
There are bandwidth settings for ethernet and wlan links. For cpu/ram there are tools you could try to leverage like `cpulimit` to carry out some of this yourself. But it is something that we may look into in the future when there is time.
